Add docs about using the Apollo Server plugin for embeded explorer (#179)
apollographql/docs: src/content/studio/explorer/embed-explorer.mdx

<EmbeddableExplorer />

> ## Apollo Server plugin

Reviewer: ```suggestion
## Embedding on the Apollo Server landing page
```

> As of Apollo Server [v3.8.0](https://github.com/apollographql/apollo-server/blob/main/CHANGELOG.md#v380) there is now a built-in plugin that will include the embeded Explorer for you.

Reviewer: ```suggestion
In Apollo Server [v3.8.0](https://github.com/apollographql/apollo-server/blob/main/CHANGELOG.md#v380) and later, you can use a built-in plugin to embed the Explorer directly on your server's landing page.
```

> You can use `ApolloServerPluginLandingPageLocalDefault` plugin for localhost development, or `ApolloServerPluginLandingPageProductionDefault` for use in a production environment.

Reviewer: ```suggestion
You use the `ApolloServerPluginLandingPageLocalDefault` plugin to configure the landing page in your local development environment, and you use `ApolloServerPluginLandingPageProductionDefault` for production environments:
```

```js
const {
 ApolloServerPluginLandingPageProductionDefault,
 ApolloServerPluginLandingPageLocalDefault
} = require('apollo-server-core');

let plugins = [];
if (process.env.NODE_ENV === 'production') {
  plugins = [ApolloServerPluginLandingPageProductionDefault({ embed: true, graphRef: 'myGraph@prod' })]
} else {
  plugins = [ApolloServerPluginLandingPageLocalDefault({ embed: true })]
}

const server = new ApolloServer({
    typeDefs,
    resolvers,
    plugins
});
```

> 

Reviewer: ```suggestion

For more information, see the [landing page plugin API reference](/apollo-server/api/plugin/landing-pages/#default-behavior).

```

> ## Custom setup

Reviewer: ```suggestion
## Embedding on an arbitrary webpage
```

Reviewer: Can we also change the "Embedding with a default operation" heading down the page to "Setting a default operation" (for less visual overlap with this modified heading)
